velox/exec/tests/HashJoinTest.cpp

```diff
@@ -844,6 +844,23 @@ TEST_P(MultiThreadedHashJoinTest, normalizedKeyOverflow) {
       .run();
 }
 
+DEBUG_ONLY_TEST_P(MultiThreadedHashJoinTest, parallelJoinBuildCheck) {
+  std::atomic<bool> isParallelBuild{false};
+  SCOPED_TESTVALUE_SET(
+      "facebook::velox::exec::HashTable::parallelJoinBuild",
+      std::function<void(void*)>([&](void*) { isParallelBuild = true; }));
+  HashJoinBuilder(*pool_, duckDbQueryRunner_)
+      .numDrivers(numDrivers_)
+      .keyTypes({BIGINT(), VARCHAR()})
+      .probeVectors(1600, 5)
+      .buildVectors(1500, 5)
+      .referenceQuery(
+          "SELECT t_k0, t_k1, t_data, u_k0, u_k1, u_data FROM t, u WHERE t_k0 = u_k0 AND t_k1 = u_k1")
+      .injectSpill(false)
+      .run();
+  ASSERT_EQ(numDrivers_ == 1, !isParallelBuild);
```

Comment:
The test checks that hash join build is not parallel. Should we add another test to check for parallel build?

Reply:
It tests both as numDrivers_ could be 1 and 3.